src/System.Private.Xml/src/System/Xml/Base64Encoder.cs

```diff
@@ -13,7 +13,7 @@ internal abstract partial class Base64Encoder
         private int _leftOverBytesCount;
         private char[] _charsLine;
 
-        internal const int Base64LineSize = 76;
+        internal const int Base64LineSize = 1024;
```

Comment:
For XML? I don't believe it's emitting a new line character after every 76 characters as it is.

---
In reply to: [122761923](https://github.com/dotnet/corefx/pull/21155#discussion_r122761923) [](ancestors = 122761923)